src/creator/loggerCreator.js

```diff
@@ -0,0 +1,49 @@
+const { createLogger, format, transports } = require("winston");
+
+function _createLogger(botConfig){
+  let logger = createLogger({
+    level: "info",
+    format: format.combine(
+      format.timestamp({
+        format: "DD.MM.YYYY HH:mm:ss",
+      }),
+      format.errors({
+        stack: true,
+      }),
+      format.splat(),
+      format.json()
+    ),
+    defaultMeta: {
+      service: botConfig.name,
+    },
+    transports: [
```

Comment:
brauchen wir wirklich 3 verscheidene logfiles? Und gibt es vlt ne möglichkeit, dass winston alte logs löscht?

Reply:
Anscheinend hiermit: 
https://github.com/winstonjs/winston-daily-rotate-file
Wollen wir das wirklich damit noch aufblähen? 
Es sind 3 verschiedene Logfiles pro Bot, ich kann gerne nur die combined behalten, ist wahrscheinlich der wichtigste. Und error behalte ich auch erstmal, info ist nebensächlich.

Reply:
Wenn du combined und error behältst, kannst du auch genauso gut nur info und error behalten. Oder alternativ nur combined. Ich finde es nicht nötig den selben log an zwei Stellen zu speichern

Reply:
Nein, error ist der wichtigste Log von allen, wenn was fehlschlägt will man das natürlich sofort sehen und nicht erstmal in den combined.log gehen und nach "error" suchen. Info als eigenständigen Log zu haben ist komplett sinnbefreit.